pom.xml

```diff
@@ -55,6 +55,16 @@
   </pluginRepositories>
 
   <dependencies>
+    <dependency>
+      <groupId>com.squareup.okhttp3</groupId>
+      <artifactId>okhttp</artifactId>
+      <version>5.0.0-alpha.11</version>
+    </dependency>
```

Comment:
Ignore that this is in here twice - we have Anda and I working on the same PR so just caught in the crossfire. We are removing as moving to http client shortly so ignore for now 🙂